deps/tls/common.lua

```diff
@@ -245,8 +245,9 @@ function TLSSocket:_read(n)
       return self:destroy(err)
     elseif cipherText then
       if self.inp:write(cipherText) then
+        local plainText
         repeat
-          local plainText = self.ssl:read()
+          plainText, extra = self.ssl:read()
```

Comment:
I don't understand this fix.  The new `extra` variable isn't declared anywhere and so we're just setting to a global.  Also the `repeat` .. `until` block in lua can see variables from inside the loop in it's conditional.  There is no need to move the local to the outside.